src/helpers/qownnotesmarkdownhighlighter.h

```diff
@@ -54,7 +54,7 @@ class QOwnNotesMarkdownHighlighter : public MarkdownHighlighter {
    private:
     Sonnet::WordTokenizer *wordTokenizer;
     Sonnet::LanguageFilter *languageFilter;
-    QOwnSpellChecker *spellchecker;
+    QOwnSpellChecker* spellchecker;
```

Comment:
clang-formatter will not like that :laughing:

Reply:
😅

Reply:
`./scripts/clang-format-project.sh` is your friend :wink:

Reply:
It will take a while getting used to doing the extra step before the commit